{{cookiecutter.project_slug}}/backend/app/app/crud/base.py

```diff
@@ -0,0 +1,218 @@
+
+                def create(self, db_session: Session, *, obj_in: schemas_item.ItemCreate) -> models_item.Item:
+                    return super(CrudItem, self).create(db_session, obj_in=obj_in)
+
+                def update(self, db_session: Session, *, obj: models_item.Item, obj_in: schemas_item.ItemUpdate) -> models_item.Item:
+                    return super(CrudItem, self).update(db_session, obj=obj, obj_in=obj_in)
+
+
+            crud_item = CrudItem(models_item.Item, schemas_item.Item)
+
+        Arguments:
+            model {Base} -- Class of the DB model which CRUD methods will be provided for
+        """  # noqa
+        self.model = model
+        self.schema = schema
+
+    def get(self, db_session: Session, obj_id: int) -> Optional[Base]:
+        """
+        get returns the object from the Database that matches the given obj_id
+        
+        Arguments:
+            db_session {Session} -- Dependency injection of the Database session, which will be used to commit/rollback changes.
+            obj_id {int} -- ID of the object in the Database. It must be defined by a PrimaryKey on the 'id' column.
+        
+        Returns:
+            Optional[Base] -- Returns an instance of self.model class if an object is found in the Database for the given obj_id. Returns None if there is no match found.
+        """  # noqa
+        return db_session.query(self.model).filter(self.model.id == obj_id).first()
+
+    def get_first_by(self, db_session: Session, **kwargs: Any) -> Optional[Base]:
+        """
+        get_by provides extended filtering capabilities: it returns the first object that matches all given **kwargs
+        
+        Arguments:
+            db_session {Session} -- Dependency injection of the Database session, which will be used to commit/rollback changes.
+        
+        Keyword Arguments:
+            kwargs {dict} -- filters formated as {attribute_name: attribute_value}
+        
+        Returns:
+            Optional[Base] -- Returns an instance of self.model class if an object is found in the Database. Returns None if there is no match found.
+        """  # noqa
+        return db_session.query(self.model).filter_by(**kwargs).first()
+
+    def get_multi(self, db_session: Session, *, skip=0, limit=100) -> List[Optional[Base]]:
+        """
+        get_multi queries all Database rows, without any filters, but with offset and limit options (for pagination purpose)
+        
+        Arguments:
+            db_session {Session} -- Dependency injection of the Database session, which will be used to commit/rollback changes.
+        
+        Keyword Arguments:
+            skip {int} -- Number of rows to skip from the results (default: {0})
+            limit {int} -- Maximum number of rows to return (default: {100})
+        
+        Returns:
+            List[Optional[Base]] -- Array of DB instances according given parameters. Might be empty if no objets are found.
```

Comment:
Typo in "objets". There are a few others, I'd suggest a spell-checker.

Reply:
Thanks for the suggestion. spell-checker installed 💯